internal/zenduty/zenduty_test.go

```diff
@@ -0,0 +1,144 @@
+package zenduty
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	ics "github.com/arran4/golang-ical"
+	"github.com/stretchr/testify/require"
```

Comment:
Adding this for three checks feels a bit much. I like https://github.com/matryer/is, but wouldn't use it here either.

Reply:
I think this is more of a habit. I always tend to use `require` in tests as it makes them shorter and I like the API of that package.